src/main/kotlin/com/openlattice/shuttle/Shuttle.kt

```diff
@@ -218,7 +218,11 @@ fun main(args: Array<String>) {
     }
 
     environment = if (cl.hasOption(ENVIRONMENT)) {
-        RetrofitFactory.Environment.valueOf(cl.getOptionValue(ENVIRONMENT).toUpperCase())
+        val env = cl.getOptionValue(ENVIRONMENT).toUpperCase()
```

Comment:
the environment theoretically doesn't need to be set as a command line argument so this might not always be caught, but I don't think anyone other than me/Todd is writing non-yaml integrations so 🤷‍♀

Reply:
I super don't see where that can happen in the code. There's not some magic environment variabling that has no code traces is there? 😨

Reply:
haha no, environment can be passed to MissionControl (which instantiates ApiClient)

Reply:
^which i guess is what happens here but MissionControl can also be called directly

Reply:
Ah yeah I gotcha. I'll add a check there too

Reply:
cool cool, yeah you can probably remove this one since it'll get caught in MissionControl but lgtm